src/app/[locale]/_component/selectIdp/idpList.tsx

```diff
@@ -21,7 +25,21 @@ type SpidLevelL3 = {
 export type SpidLevels = SpidLevelL1 | SpidLevelL2 | SpidLevelL3;
 
 export function IdpList({ spidLevel }: IdpList) {
+  const { userLogged } = useLogin();
+  const token = storageTokenOps.read();
+  const taxCode = token ? userFromJwtToken(storageTokenOps.read()).taxCode : undefined;
+
+  const savePrivilegesData = () => {
+    if (taxCode) {
+      storagePrivilegeOps.write({
+        previous: userLogged?.spidLevel || undefined,
```

Comment:
Is this (`previous`) intentionally named in a cryptic way, or we could give it a more meaningful name (e.g. `previousSecurityLevel`, `previousAccessLevel` or something like that)?

Reply:
corrected